docs/rules/require-jsdoc.md

```diff
@@ -0,0 +1,56 @@
+# Require JSDoc comment (require-jsdoc)
+
+This rule generates warnings for nodes which do not have JSDoc comments. It considered a good practice to document the behavior of different nodes to help engineers understand the functionality of the node.
+Supported nodes:
+
+* `FunctionExpression`
```

Comment:
Yeah, but I'd like to turn this on for ESLint, so I want to know how it works on our codebase. :)